PowerUp/app/src/androidTest/java/powerup/systers/com/powerup/espresso/AvatarRoomActivityTest.java

```diff
@@ -0,0 +1,226 @@
+        onView(withId(R.id.face)).check(matches(new DrawableMatcher(R.drawable.face3)));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
+        onView(withId(R.id.face)).check(matches(new DrawableMatcher(R.drawable.face3)));
+        onView(withId(R.id.faceLeft)).perform(click());
+        onView(withId(R.id.face)).check(matches(new DrawableMatcher(R.drawable.face2)));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+        onView(withId(R.id.faceLeft)).perform(click());
+        onView(withId(R.id.face)).check(matches(new DrawableMatcher(R.drawable.face1)));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
+        onView(withId(R.id.faceLeft)).perform(click());
+        onView(withId(R.id.face)).check(matches(new DrawableMatcher(R.drawable.face3)));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+        onView(withId(R.id.faceLeft)).perform(click());
+        onView(withId(R.id.face)).check(matches(new DrawableMatcher(R.drawable.face2)));
+        onView(withId(R.id.faceRight)).perform(click());
+        onView(withId(R.id.face)).check(matches(new DrawableMatcher(R.drawable.face3)));
+
+
+        onView(withId(R.id.clotheLeft)).perform(click());
+        onView(withId(R.id.clothes)).check(matches(new DrawableMatcher(R.drawable.cloth4)));
+        onView(withId(R.id.clotheLeft)).perform(click());
+        onView(withId(R.id.clotheRight)).perform(click());
+        onView(withId(R.id.clothes)).check(matches(new DrawableMatcher(R.drawable.cloth4)));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
+        onView(withId(R.id.clothes)).check(matches(new DrawableMatcher(R.drawable.cloth4)));
+        onView(withId(R.id.clotheLeft)).perform(click());
+        onView(withId(R.id.clothes)).check(matches(new DrawableMatcher(R.drawable.cloth3)));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+        onView(withId(R.id.clotheLeft)).perform(click());
+        onView(withId(R.id.clothes)).check(matches(new DrawableMatcher(R.drawable.cloth2)));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
+        onView(withId(R.id.clotheLeft)).perform(click());
+        onView(withId(R.id.clothes)).check(matches(new DrawableMatcher(R.drawable.cloth1)));
+        activityRule.getActivity().setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
+        onView(withId(R.id.clotheLeft)).perform(click());
+        onView(withId(R.id.clothes)).check(matches(new DrawableMatcher(R.drawable.cloth4)));
+        onView(withId(R.id.clotheRight)).perform(click());
+        onView(withId(R.id.clothes)).check(matches(new DrawableMatcher(R.drawable.cloth1)));
```

Comment:
In this entire method you are testing the current setting with the current resources for clothes, face, etc. However, these can be changed or increased at any time. Therefore instead create a list with the elements we currently have so if they are ever changed, the test can be easily adapted. Also don't forget to test the head. Even if there is just one option now, there could be more in the future